diff -aurN systemd-186.orig/src/journal/journald.conf systemd-186/src/journal/journald.conf
--- systemd-186.orig/src/journal/journald.conf	2012-07-03 22:09:34.129764462 +0200
+++ systemd-186/src/journal/journald.conf	2012-07-03 22:09:50.366561475 +0200
@@ -22,7 +22,7 @@
 #ForwardToSyslog=yes
 #ForwardToKMsg=no
 #ForwardToConsole=no
-#ImportKernel=no
+ImportKernel=no
 #TTYPath=/dev/console
 #MaxLevelStore=debug
 #MaxLevelSyslog=debug
