diff -aurN systemd-183.orig/src/core/machine-id-setup.c systemd-183/src/core/machine-id-setup.c
--- systemd-183.orig/src/core/machine-id-setup.c	2012-05-25 15:24:20.749096009 +0200
+++ systemd-183/src/core/machine-id-setup.c	2012-05-26 12:45:35.075397136 +0200
@@ -161,9 +161,10 @@
         /* We create this 0444, to indicate that this isn't really
          * something you should ever modify. Of course, since the file
          * will be owned by root it doesn't matter much, but maybe
-         * people look. */
+         * people look.
+         * ADDED BY Freddix: use 0644, 0444 causes troubles here */
 
-        fd = open("/etc/machine-id", O_RDWR|O_CREAT|O_CLOEXEC|O_NOCTTY, 0444);
+        fd = open("/etc/machine-id", O_RDWR|O_CREAT|O_CLOEXEC|O_NOCTTY, 0644);
         if (fd >= 0)
                 writable = true;
         else {
