diff -aurN systemd-187.orig/configure.ac systemd-187/configure.ac
--- systemd-187.orig/configure.ac	2012-07-30 00:31:42.778698704 +0200
+++ systemd-187/configure.ac	2012-07-30 01:05:53.802698486 +0200
@@ -587,6 +587,12 @@
                 AC_DEFINE(TARGET_MAGEIA, [], [Target is Mageia])
                 M4_DEFINES=-DTARGET_MAGEIA=1
                 ;;
+	freddix)
+	        SYSTEM_SYSVINIT_PATH=
+		SYSTEM_SYSVRCND_PATH=
+		AC_DEFINE(TARGET_FREDDIX, [], [Target is Freddix])
+		M4_DISTRO_FLAG=-DTARGET_FREDDIX=1
+                ;;
         other)
                 ;;
         *)
