diff -aurN systemd-209.orig/src/libsystemd/sd-event/sd-event.c systemd-209/src/libsystemd/sd-event/sd-event.c
--- systemd-209.orig/src/libsystemd/sd-event/sd-event.c	2014-02-21 11:46:01.792934587 +0100
+++ systemd-209/src/libsystemd/sd-event/sd-event.c	2014-02-21 11:47:29.850474239 +0100
@@ -771,7 +771,7 @@
                                    sd_event_time_handler_t callback,
                                    void *userdata) {
 
-        return event_add_time_internal(e, ret, SOURCE_REALTIME, &e->realtime_fd, CLOCK_REALTIME, &e->realtime_earliest, &e->monotonic_latest, usec, accuracy, callback, userdata);
+        return event_add_time_internal(e, ret, SOURCE_REALTIME, &e->realtime_fd, CLOCK_REALTIME, &e->realtime_earliest, &e->realtime_latest, usec, accuracy, callback, userdata);
 }
 
 static int event_update_signal_fd(sd_event *e) {
